src/AlgebraicPetri.jl

```diff
@@ -7,6 +7,7 @@ export SchPetriNet, PetriNet, OpenPetriNetOb, AbstractPetriNet, ns, nt, ni, no,
   add_species!, add_transition!, add_transitions!,
   add_input!, add_inputs!, add_output!, add_outputs!, inputs, outputs,
   TransitionMatrices, vectorfield,
+  LabelledPetriNetUntyped,
```

Comment:
```suggestion
```

I don't think we want to expose this type out, we can still access it in the `src/OpenTransitions.jl` file with a `using`. I will add a comment there as well.